server/index.js

```diff
@@ -51,6 +50,37 @@ app.get('/api/notes', (req, res, next) => {
     .catch(err => next(err));
 
 });
+
+app.delete('/api/recordings/:id', (req, res, next) => {
+  const { id } = req.params;
+  if (!id) {
+    throw new ClientError(400, 'bad request');
+  }
+
+  // delete from "recordings"
+
+  const params = [id];
+  const sql = `
+  delete from "recordings"
+  where "recordingId" = $1
+  returning *
+  `;
+  db.query(sql, params)
+    .then(result => {
+      if (result.rows[0]) {
+        const filePath = path.join(__dirname, `/public${result.rows[0].url}`);
+        fs.unlink(filePath, err => {
+          if (err) {
+            console.error(err);
+          }
+        });
+        res.status(204).json();
```

Comment:
You don't want to respond unless you know this succeeded (move response into the callback). If there is an error, pass it to `next()`.